RELEASE.md

```diff
@@ -126,13 +126,13 @@ git push origin "${tag}" "pkg/apis/monitoring/${tag}" "pkg/client/${tag}"
 
 Signed tag with a GPG key is appreciated, but in case you can't add a GPG key to your Github account using the following [procedure](https://docs.github.com/articles/generating-a-gpg-key), you can replace the `-s` flag by `-a` flag of the `git tag` command to only annotate the tag without signing.
 
-Once a tag is created, the `publish` Github action will push the container images to [quay.io](https://quay.io/organization/prometheus-operator) and [ghcr.io](https://github.com/prometheus-operator/prometheus-operator/pkgs/container/prometheus-operator). Wait until the [publish](https://github.com/prometheus-operator/prometheus-operator/actions/workflows/publish.yaml) workflow is complete before going to the next step.
+Once a tag is created, the `publish` Github action will push the container images to [quay.io](https://quay.io/organization/prometheus-operator) and [ghcr.io](https://github.com/rhobs/obo-prometheus-operator/pkgs/container/prometheus-operator). Wait until the pulbish workflow is complete before going to the next step.
```

Comment:
same here:
changed here url because script was failing:
```
            RELEASE.md:98: "https://github.com/rhobs/obo-prometheus-operator/compare/v0.72.0...release-0.73" not accessible; status code 404: Not Found
                RELEASE.md:129: "https://github.com/rhobs/obo-prometheus-operator/actions/workflows/publish.yaml" not accessible; status code 404: Not Found
```